tests/test_pipeline.py

```diff
@@ -55,3 +91,19 @@ def test_image_to_text_vit(self):
         model = "nlpconnect/vit-gpt2-image-captioning"
         expected_result = "a soccer game with a player jumping to catch"
         self._test_image_to_text(model, expected_result)
+
+    def test_text_to_speech_speecht5(self):
+        model = "microsoft/speecht5_tts"
+        expected_result = 16000
+        self._test_text_to_speech(model, expected_result)
+
+    def test_text_to_speech_m4t(self):
+        model = "facebook/hf-seamless-m4t-medium"
+        expected_result = 16000
+        if packaging.version.parse(transformers.__version__) >= packaging.version.parse("4.40.0"):
+            self._test_text_to_speech(model, expected_result)
+
+    def test_text_to_speech_mms(self):
+        model = "facebook/mms-tts-eng"
+        expected_result = 16000
+        self._test_text_to_speech(model, expected_result)
```

Comment:
We can have only one method called `test_text_to_speech` with the `@pytest.mark.parametrize(...)` decorator to avoid defining one test per model. Here is an example: https://github.com/huggingface/optimum-habana/blob/574702cd6b4e755cb3673806671352c0e6d3957b/tests/test_text_generation_example.py#L245